synapse/crypto/keyring.py

```diff
@@ -279,6 +282,11 @@ async def process_request(self, verify_request: VerifyJsonRequest) -> None:
 
         key_ids_to_find = set(verify_request.key_ids) - found_keys.keys()
         if key_ids_to_find:
+            # We're still missing some keys. Consult each of our `KeyFetcher` instances
+            # (stored in `self._key_fetchers`) to try and find them.
+            # Key fetch attempts are queued via `self._server_queue` below, and carried
+            # out in `self._inner_fetch_key_requests`.
```

Comment:
Most of this feels like it should be in the docstring to `add_to_queue`? (or the `_inner_fetch_key_request` or `Keyring` or `_key_fetchers` or something?)

I think putting it here will bury it.

Reply:
I thought about that, though in practice at this bit in the codebase I saw:

```
found_keys_by_server = await self._server_queue.add_to_queue(
    key_request, key=verify_request.server_name
)
```

and assumed that keys were being fetched from the remote server. Having this comment here would have helped. We could potentially rename `_server_queue` to `_fetch_keys_queue` as well/instead.

Reply:
I think renaming server queue is likely the way to go? And maybe move the comment to where it is declared?